packages/adapter-auto/package.json

```diff
@@ -11,7 +11,8 @@
 	"type": "module",
 	"exports": {
 		".": {
-			"import": "./index.js"
+			"import": "./index.js",
+			"types": "./index.d.ts"
```

Comment:
Think we should flip `import` and `types` so `types` comes first so it's more compliant to the resolve algorithm. The Svelte repo also did the same.

Reply:
Yes, types should be first

Reply:
fixed